Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -485,8 +486,35 @@ namespace gdjs {
       point[1] = position.y;
       return point;
     }
+    applyAntialiasing(): void {
+      if (this._object.isAntialiasingOn()) {
+        let antialiasingFilter = new PIXI.filters.FXAAFilter();
+        antialiasingFilter.enabled = true;
+        switch (this._object.getAntialiasingQuality()) {
+          case 2:
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.LOW;
+            break;
+          case 4:
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.MEDIUM;
+            break;
+          case 8:
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.HIGH;
+            break;
+          default:
+            //Should never be executed but if it's not a valid value, set the quality to LOW.
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.LOW;
+
+            break;
```

Comment:
```suggestion
            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.LOW;
            break;
```

Generally, comments should only be used to describe something that you fail to describe directly with your code. This comment doesn't explain anything that we cannot see directly from the code: "when none of the other cases have been met use a low anti-aliasing" - we can see that through the `default` case and the setting of `antialiasingFilter.multisample` to `LOW`. 

The part that says that this code path should be unreachable is indeed something that is not expressed by your code, but not really relevant to the implementation. It doesn't add anything except incentive to ignore this part of code when doing other changes, which can be dangerous if for some reason the code in `RuntimeObject` changes to no longer guarantee the value to be 2, 4 or 8, and this code path becomes the only guard that prevents an invalid value, and people seeing this comment think that it is an unreachable code path when it is not, this can lead to hard to understand situations for another developer, because they've been mislead by an out of date comment.

tl;dr if your comment is not absolutely necessary to understand your code, or refers to another part of the codebase (that may change, making the comment outdated), you should probably leave it out as it only makes the code more difficult to read otherwise, which is the point of comments